meson_options.txt

```diff
@@ -142,6 +142,10 @@ option('plugin_acpi_phat',
     'false': 'disabled',
   },
 )
+option('plugin_algoltekusb',
```

Comment:
Most (all?) USB plugins don't get their own option to enable or disable. We pretty much key off just gusb to decide whether to build or not.

So I think you can probably drop this hunk.

Reply:
Removed this part in meson_options.txt.